kibicara/platforms/email/bot.py

```diff
@@ -0,0 +1,56 @@
+# Copyright (C) 2020 by Maike <maike@systemli.org>
+#
+# SPDX-License-Identifier: 0BSD
+
+from kibicara.platforms.email.model import EmailSubscribers, Email
+from kibicara.platformapi import Censor, Spawner
+from kibicara.email import send_email
+from kibicara.config import config
+from kibicara.webapi.admin import to_token
+from smtplib import SMTPException
+from logging import getLogger
+
+
+logger = getLogger(__name__)
+
+
+class EmailBot(Censor):
+    def __init__(self, email_model):
+        super().__init__(email_model.hood)
+        self.model = email_model
+        self.messages = []
+
+    async def run(self):
+        """ Loop which waits for new messages and sends emails to all subscribers. """
+        while True:
+            message = await self.receive()
+            for subscriber in EmailSubscribers(hood=self.hood.id):
```

Comment:
I don't think this fetches from the database, just constructs a `EmailSubscribers` object.
Should be something like `EmailSubscribers.objects.filter(hood=self.hood).all()`